src/V3Case.cpp

```diff
@@ -227,15 +228,51 @@ class CaseVisitor final : public VNVisitor {
                 }
             }
         }
-        for (uint32_t i = 0; i < numCases; ++i) {
-            if (!m_valueItem[i]) {
-                nodep->v3warn(CASEINCOMPLETE, "Case values incompletely covered "
-                                              "(example pattern 0x"
-                                                  << std::hex << i << ")");
-                m_caseNoOverlapsAllCovered = false;
-                return false;
+        if (AstEnumDType* enumDtp = VN_CAST(nodep->exprp()->dtypep()->skipRefToEnump(), EnumDType)) { // CASE is enum
+            if (nodep->uniquePragma() || nodep->priorityPragma()) {
+              suppressCaseIncWarn =true; // suppress the warning when we have unique or priority
+            } else { // check if all values were supplied
+                 bool foundMatch =false;
+                 for (uint32_t i = 0; i < numCases; ++i) { // iterate through all case items
+                     for (AstEnumItem* itemp = enumDtp->itemsp(); itemp;
+                                       itemp = VN_AS(itemp->nextp(), EnumItem)) { //iterate through all enum values
```

Comment:
This is O(n^2).  Instead make a std::set, put all case items into it, then make another set and put enum items into it, then check sets are equal - C++ does the hard work you can just run (caseSet == enumSet).

Reply:
I fixed all the other issues. Would you mind taking care of this one?